### ISSUE TITLE
fetch_apple subtest started to fail

### ISSUE THREAD

**Issue by eserte** [briandfoy/net-mac-vendor#26]
t/fetch_oui.t fails, probably because of changed capitalization on the service side:
```
    ok 1 - A reference of type 'ARRAY' isa 'ARRAY'
    ok 2 - Line 0 matches for 00-0D-93
    ok 3 - Line 1 matches for 00-0D-93
    not ok 4 - Line 2 matches for 00-0D-93
    #   Failed test 'Line 2 matches for 00-0D-93'
    #   at t/fetch_oui.t line 50.
    #          got: 'Cupertino  CA  95014'
    #     expected: 'CUPERTINO  CA  95014'
    ok 5 - Line 3 matches for 00-0D-93
    1..5
    # Looks like you failed 1 test of 5.
not ok 3 - fetch_apple
```

**Comment by OneLoveAmaru** [comment]
Same here, how can I fix?

    #          got: 'Cupertino  CA  95014'
    #     expected: 'CUPERTINO  CA  95014'
    # Looks like you failed 1 test of 5.

**Comment by OneLoveAmaru** [comment]
I ended just just running "cpan -f install NAME" and got it installed.

**Comment by karenetheridge** [comment]
This is fixed in 1.263 and this issue can now be closed.

**Comment by eserte** [comment]
This particular issue looks fixed.
(However, the ieee server seems often to respond with 500 errors, so the test matrix doesn't look good with the new release)
